internal/charmstore/migrations_test.go

```diff
@@ -371,10 +408,13 @@ func (s *migrationsSuite) TestDenormalizeEntityIdsSomeUpdates(c *gc.C) {
 		Size:     1,
 	})
 	s.checkEntity(c, &mongodoc.Entity{
-		URL:     id2,
-		BaseURL: baseURL(id2),
-		Name:    "django",
-		Size:    2,
+		URL:      id2,
+		BaseURL:  baseURL(id2),
+		Name:     "django",
+		User:     "",
```

Comment:
I'm not sure why @frankban might know but It checks that we don't update those fields if name is already set.